### PR TITLE
vyos logging_global resource module

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

Logging resource module vyos_logging_global

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

vyos_logging_global

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by KB-perByte** [comment]
<div id="header" style="background: rgb(248, 248, 248); width: 1920px; border-bottom: 1px solid rgb(238, 238, 238); color: rgb(0, 0, 0); font-family: georgia, serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="content" style="padding: 1em 3em;"><h1 style="margin: 0px; padding: 0px; border: 0px; outline: 0px; font-weight: inherit; font-style: inherit; font-size: 1.25em; font-family: inherit; vertical-align: baseline; display: inline-block;">Coverage report:<span> </span><span class="pc_cov">76%</span></h1>Show keyboard shortcuts<form id="filter_container" style="display: inline-block; float: right; margin: 0px 2em 0px 0px;"><input id="filter" type="text" value="" placeholder="filter..." style="width: 10em;"></form></div></div><div id="index" style="margin: 1em 0px 0px 3em; color: rgb(0, 0, 0); font-family: georgia, serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Module | statements | missing | excluded | branches | partial | coverage
-- | -- | -- | -- | -- | -- | --
Total | 311 | 2 | 0 | 138 | 7 | 98%
plugins/module_utils/network/vyos/argspec/logging_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/argspec/logging_global/logging_global.py | 4 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/config/logging_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/config/logging_global/logging_global.py | 90 | 0 | 0 | 64 | 1 | 99%
plugins/module_utils/network/vyos/facts/logging_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/facts/logging_global/logging_global.py | 54 | 1 | 0 | 32 | 0 | 99%
plugins/module_utils/network/vyos/rm_templates/logging_global.py | 52 | 0 | 0 | 40 | 5 | 95%
plugins/modules/vyos_logging_global.py | 14 | 1 | 0 | 2 | 1 | 88%
tests/unit/modules/network/vyos/test_vyos_logging_global.py | 97 | 0 | 0 | 0 | 0 | 100%

</div>

**Comment by KB-perByte** [comment]
Module | statements | missing | excluded | branches | partial | coverage
-- | -- | -- | -- | -- | -- | --
Total | 336 | 2 | 0 | 150 | 8 | 98%
plugins/module_utils/network/vyos/argspec/logging_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/argspec/logging_global/logging_global.py | 4 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/config/logging_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/config/logging_global/logging_global.py | 122 | 0 | 0 | 88 | 2 | 99%
plugins/module_utils/network/vyos/facts/logging_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/vyos/facts/logging_global/logging_global.py | 51 | 1 | 0 | 30 | 1 | 98%
plugins/module_utils/network/vyos/rm_templates/logging_global.py | 47 | 0 | 0 | 30 | 4 | 95%
plugins/modules/vyos_logging_global.py | 14 | 1 | 0 | 2 | 1 | 88%
tests/unit/modules/network/vyos/test_vyos_logging_global.py | 98 | 0 | 0 | 0 | 0 | 100%

**Comment by KB-perByte** [comment]
recheck

File: plugins/modules/vyos_logging_global.py [hunk 1]
```diff
@@ -0,0 +1,756 @@
...
+     of the parameters above.
+  type: dict
+after:
+  description: The resulting configuration model invocation.
+  returned: when changed
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+  type: dict
+commands:
```
Comment (NilashishC) [inline plugins/modules/vyos_logging_global.py:709]: Not all return values are added here. This should be generated by cli_rm_builder.

**Review by NilashishC — APPROVED** [review]

**Pull request merged**